### ISSUE TITLE
Add option to force a width of the urlbar dropdown.

### ISSUE THREAD

**Issue by RobertZenz** [RobertZenz/ThinBars#3]
Sometimes there is only limited horizontal space available, so forcing the urlbar dropdown to a certain size would help in these situations tremendously.

**Comment by RobertZenz** [comment]
Something like the following should suffice:

```
#PopupAutoCompleteRichResult,
#PopupAutoCompleteRichResult > * {
    min-width: 800px !important;
}
```

**Comment by RobertZenz** [comment]
Implemented in 6f3a1a6.
